submissions/Harnytskyi/friends-app/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Friends App</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,500;1,400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+</head>
+<body>
+    <header class="header">
+            <div class="header-title"><b>Friends</b>App</div>
+        <input id="search" class="search" onkeyup="search()" placeholder="Search" onsearch="search()" type="search">
+    </header>
+    <main>
+        <div class="filters">
+            <div class="filter-item">
+                <p class="filters-header"><b>Sort</b></p>
+                <input type="button" class="button-sort" id="a-z" value="A-Z">
+                <input type="button" class="button-sort" id="z-a" value="Z-A">
+                <br>
+                <input type="button" class="button-sort" id="0-9" value="0-9">
+                <input type="button" class="button-sort" id="9-0" value="9-0">
```

Comment:
Just make it as list of radio buttons. Then HTML will hold a state of current selected input and you will have a possibility just to get `change` event with `value` of checked element.

Reply:
replaced these `button` with `radio`